corehq/apps/reports/util.py

```diff
@@ -434,7 +434,7 @@ def stream_qs(qs, batch_size=1000):
         for item in qs:
             yield item
 
-def numcell(text, value=None, convert='int'):
+def numcell(text, value=None, convert='int', raw=None):
```

Comment:
We shouldn't probably override like this, make this more extensible from the caller instead (caller passes formatter & unformatter) and freeze all of this into API.